clients/nimbus-el/Dockerfile

```
@@ -1,36 +1,39 @@
 # Docker container spec for building the master branch of nimbus.
 
-FROM debian:buster-slim AS build
+FROM debian:testing-slim AS build
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get clean && apt update \
+ && apt -y install curl build-essential git-lfs librocksdb-dev
+
+RUN ldd --version ldd
```

```suggestion
RUN ldd --version
```
Maybe a typo? It returns the same output for me without the last keyword.